### PR TITLE
[RHPAM-4140-7.13] Hide adminPassword from Business automation operator kieapp instance

### PR DESCRIPTION
**Pull request by desmax74**
See: https://issues.redhat.com/browse/RHPAM-4140
Signed-off-by: desmax74 <mdessi@redhat.com>

### CODE REVIEW

**Comment by desmax74** [comment]
@spolti @jakubschwan @davidesalerno

File: pkg/controller/kieapp/defaults/defaults.go [hunk 1]
```diff
@@ -537,6 +530,8 @@ func getConsoleTemplate(cr *api.KieApp) api.ConsoleTemplate {
 		if cr.Status.Applied.Environment == api.RhpamAuthoringHA || cr.Status.Applied.Environment == api.RhdmAuthoringHA {
 			if cr.Status.Applied.Objects.Console.DataGridAuth != nil {
 				template.DataGridAuth = *cr.Status.Applied.Objects.Console.DataGridAuth
+			} else {
+				template.DataGridAuth = api.DataGridAuth{}
```
Comment (jakubschwan) [inline pkg/controller/kieapp/defaults/defaults.go:534]: This change is not included in PR for changes on main branch - see #779

File: pkg/controller/kieapp/defaults/defaults.go [hunk 2]
```diff
@@ -1529,7 +1543,7 @@ func SetDefaults(cr *api.KieApp) {
 				}
 			} else {
 				//if not provided we create default credentials to allow the correct deployment of the HA env
-				specApply.Objects.Console.DataGridAuth = &api.DataGridAuth{Username: constants.DefaultDatagridUsername, Password: string(shared.GeneratePassword(8))}
+				specApply.Objects.Console.DataGridAuth = &api.DataGridAuth{Username: constants.DefaultDatagridUsername, Password: ""}
```
Comment (jakubschwan) [inline pkg/controller/kieapp/defaults/defaults.go:1546]: left over from debugging?

File: pkg/controller/kieapp/defaults/defaults.go (hunk 1 above)
Comment (desmax74) [reply pkg/controller/kieapp/defaults/defaults.go:534]: Fixed

File: pkg/controller/kieapp/defaults/defaults.go (hunk 2 above)
Comment (desmax74) [reply pkg/controller/kieapp/defaults/defaults.go:1554]: No, leaved empty to check later the len

**Comment by desmax74** [comment]
/@davidesalerno

**Review by davidesalerno — APPROVED** [review]

**Pull request merged**